src/main/java/com/daitangroup/vehiclesTypes/UtilityVehicle.java

```diff
@@ -0,0 +1,238 @@
+        } else {
+            LOGGER.warn("No more gears left to change");
+        }
+        return actualGear;
+    }
+
+    public void putOnNeutral() {
+        actualGear = 0;
+        System.out.println("Putting on neutral");
+    }
+
+    /**
+     * Class constructor
+     * @param fuel Fuel of vehicle in liters
+     * @param numberOfGears Number of shift gears
+     * @param loadCapacity Back load capacity
+     * @param numberOfDoors Numeber of doors in the vehicle
+     */
+    public UtilityVehicle(float fuel, int numberOfGears, float loadCapacity, int numberOfDoors, int maxNumberOfOccupants) {
+        this.fuel = fuel;
+        this.numberOfGears = numberOfGears;
+        this.loadCapacity = loadCapacity;
+        this.numberOfDoors = numberOfDoors;
+        this.maxNumberOfOccupants = maxNumberOfOccupants;
+        this.kindOfLoads = new ArrayList<String>();
+        this.actualGear = 0;
+    }
+
+
+    //getters and setters
+    public float getFuel() {
+        return fuel;
+    }
+
+    public void setFuel(float fuel) {
+        this.fuel = fuel;
+    }
+
+    public int getNumberOfGears() {
+        return numberOfGears;
+    }
+
+    public void setNumberOfGears(int numberOfGears) {
+        this.numberOfGears = numberOfGears;
+    }
+
+    public int getActualGear() {
+        return actualGear;
+    }
+
+    public void setActualGear(int actualGear) {
+        this.actualGear = actualGear;
+    }
+
+    public float getLoadCapacity() {
+        return loadCapacity;
+    }
+
+    public void setLoadCapacity(float loadCapacity) {
+        this.loadCapacity = loadCapacity;
+    }
+
+    public int getNumberOfDoors() {
+        return numberOfDoors;
+    }
+
+    public void setNumberOfDoors(int numberOfDoors) {
+        this.numberOfDoors = numberOfDoors;
+    }
+
+    public List<String> getKindOfLoads() {
+        return kindOfLoads;
+    }
+
+    public void setKindOfLoads(List<String> kindOfLoads) {
+        this.kindOfLoads = kindOfLoads;
+    }
+
+    public void addKindOfLoad(String load) {
+        this.kindOfLoads.add(load);
+    }
+
+    /**
+     *Check if the car is already full and try to add the occupant. If the occupant is already in the car it gets error.
+     *
+     * @param name name of occupant that will enter in the motorcycle
+     */
+    public void addOccupant(String name) {
+        if (occupantsNames.size() < maxNumberOfOccupants){
+            try {
+                occupantsNames.add(name);
+            } catch (Exception e) {
+                LOGGER.error("Fail to insert new occupant. Already in vehicle. " + e);
+            }
+        } else {
+            LOGGER.error("The vehicle is full!" + occupantsNames.size());
+        }
+    }
+
+    /**
+     * Check if has at least one occupant and try to remove. If the ocuppant is not in the motorcycle, show an error.
+     * @param name name of occupant that will exit in the motorcycle
+     */
+    public void removeOccupant(String name) {
```

Comment:
Nice to see you're documenting your methods. Even better if you document then in the interface and use this documentation as an informal contract the implementations must obey, as in general you program for interfaces and do not know the concrete class at compile time. And as @mfreitas-daitan has said, ALWAYS use the @Override annotation. This avoids a lot of pain.